karaf/features/src/main/feature/feature.xml

```diff
@@ -55,7 +55,12 @@
         <bundle>mvn:javax.annotation/javax.annotation-api/${cxf.javax.annotation-api.version}</bundle>
     </feature>
 
+    <feature name="brooklyn-config" version="${project.version}">
+        <feature>brooklyn-server-config</feature>
+    </feature>
+
     <feature name="brooklyn-headless" version="${project.version}" description="All Brooklyn bundles witht the exception of the launcher">
+        <feature prerequisite="true">brooklyn-config</feature>
```

Comment:
It does not matter here as it will self extract the files on first boot. As in, the configuration files will be installed before everything else which is what we want.